src/crypting/Main.java

```diff
@@ -0,0 +1,42 @@
+package crypting;
+
+import crypting.strategy.UnicodeCipher;
+
+public class Main {
+
+    private static CliParser parser;
+    private static Cryptor cryptor;
+
+    public static void main(String[] args) {
+        parser = new CliParser(args);
+        cryptor = new Cryptor();
+
+        if ("unicode".equals(parser.optionOf("-alg")))
```

Comment:
What happens if statement will be false?

Reply:
By the default, Cryptor uses ShiftCipher, instance of which is created inside its constructor

> When starting the program, the necessary algorithm should be specified by an argument (-alg). The first algorithm should be named shift, the second one should be named unicode. If there is no -alg you should default it to shift.

Reply:
Fixed

Reply:
What fixed?

Reply:
> What happens if statement will be false?

The default implementation of Cipher will be used.

> What fixed?

Now we explicitly set the default cipher implementation by passing ShiftCipher instance into Cryptor constructor. So there is no need to create "else" situation

Reply:
> The default implementation of Cipher will be used.
How I can know it? What happens if I will pass `baconian` as input parameter?

Reply:
Added algorithm selection condition

Reply:
I don's see your answer